cypress/e2e/group/showGroupsAndClassesInTable.feature

```diff
@@ -0,0 +1,83 @@
+@release
```

Comment:
can you change the name of the folder to -> 'class_management' instead of 'group'